cucumber/api/features/step_definitions/user_steps.rb

```diff
@@ -1,119 +1,53 @@
 # frozen_string_literal: true
 
-When(/^I am the super\-user$/) do
+When("I am the super-user") do
   @current_user = admin_user
 end
 
-When(/^I am a user named "([^"]*)"$/) do |login|
+When("I am a user named {string}") do |login|
   @current_user = create_user(login, admin_user)
 end
 
-Given(/^I create a new user "([^"]*)"$/) do |login|
+Given("I create a new user {string}") do |login|
   create_user login, @current_user || admin_user
 end
 
-Given(/^I have user "([^"]*)"$/) do |login|
+Given("I have user {string}") do |login|
   unless user_exists?(login)
     create_user login, @current_user || admin_user
   end
 end
 
-Given(/^I have host "([^"]*)"$/) do |login|
+Given("I have host {string}") do |login|
   unless host_exists?(login)
     create_host login, @current_user || admin_user
   end
 end
 
-Given(/^I create a new admin-owned user "(.*?)"$/) do |login|
+Given("I create a new admin-owned user {string}") do |login|
   create_user login, admin_user
 end
 
-Given(/^I create a new user "([^"]*)" in account "([^"]*)"$/) do |login, account|
+Given('I create a new user {string} in account {string}') do |login, account|
   roleid = "#{account}:user:#{login}"
   Role.create(role_id: roleid)
 end
 
-When(/^I operate on "([^"]*)"$/) do |login|
-  @current_user = lookup_user(login)
-end
-
-Given(/^I login as "([^"]*)"$/) do |login|
-  roleid = if login.include?(":")
-    login
-  else
-    "cucumber:user:#{login}"
-  end
-
+Given("I login as {string}") do |login|
+  roleid = (login.include?(":") ? login : "cucumber:user:#{login}")
   @current_user = Role.with_pk!(roleid)
   Credentials.new(role: @current_user).save unless @current_user.credentials
 end
 
-Given(/^I log out$/) do
+Given("I log out") do
   @current_user =  nil
+  # TODO: investigate smell
   headers.delete(:authorization) if headers.key?(:authorization)
 end
   
-When(/^I have a password$/) do
-  @current_user.password = "password"
-  @current_user.save
-end
-
-Given(/^I set the password for "([^"]*)" to "([^"]*)"$/) do |login, password|
+Given("I set the password for {string} to {string}") do |login, password|
+  # TODO: investigate smell
```

Comment:
These guys shouldn't be showing up in your PR, since they were already merged.  Maybe you need a rebase?